### DISCUSSION TITLE
Error when installing webMAN_MOD_1.47.42

### DISCUSSION THREAD

**Discussion by TheSpielor** [aldostools/webMAN-MOD#826] [Q&A]
hej there,
i have freshly installed the CFW 4.89 Evilnat CEX (Cobra 8.3) guided Modded Warefares YouTube Video "How to Jailbreak the PS3 on 4.89".
It all went fine and i went over to the next video of his playlist: Episode 2 "PS3 Jailbreak Setting up MultiMan/WebMan Mod".

MultiMan is now installed and ive downloaded webMAN-MOD here. at the installation progress it was quiet on about 46/47%. after few seconds it went further very quick to about 73% and slower progress to 87%.

at 87% the installation stops with an error: (8001002B)

ive downloaded the .pkg-file on another pc and tried again. also restart of the ps3 and different stuff...  nothing helps. :( any help???

**Comment by aldostools** [comment]
Error 8001002B = Filesystem mounting failed (actually IO error...EIO)

It seems that there is an issue with your internal HDD.
